### PR TITLE
Updates for 2022 that changes maximum year to 2021 and points queries to API rather than directly at Rasdaman.

### PR DESCRIPTION
**Pull request by BobTorgerson**
This PR modifies the HSIAA to include the new data added to the 'hsia_arctic_production' for 2018-2021. To use these updates, please use this PR for the data-api locally: https://github.com/ua-snap/data-api/pull/210

### CODE REVIEW

**Review by cstephen — CHANGES_REQUESTED** [review]
I tried running the web app's `atlas_updates_2022` branch against the `seaice_conc` branch of the API and noticed that you get an HTTP 500 error when `/hsia` is appended to the end of seaice API queries. I added this to the data-api PR review: https://github.com/ua-snap/data-api/pull/210

I'll check out the web app once the data-api issues are resolved. My hunch is that the work to solve the data-api issue was probably done but not pushed to GitHub.

File: src/App.vue [hunk 1]
```diff
@@ -812,7 +811,7 @@ export default {
         let title = this.getChartTitlePlaceFragment();
 
         this.thresholdChartLayout = {
-          title: `<b>${title}, 1850-2017</b>`,
+          title: `<b>${title}, 1850-2022</b>`,
```
Comment (brucecrevensten) [inline src/App.vue:814]: Why is this one 2022 instead of 2021 like everywhere else?

Comment (BobTorgerson) [reply src/App.vue]: That's an oversight from when I originally was going to include May 2022 but figured we'll just do all of 2022 in 2022. I've changed this in the most recent commit.

**Comment by cstephen** [comment]
The changes to use the standard sea ice concentration API endpoint look good to me, and everything appears to be working except for this:

- [ ] The CSV download file names have very long lat/lon coordinates (e.g., `Sea Ice Concentration for 67.30454201355695, -168.17499308956033`). I think this can be fixed easily enough by adding the `:download="downloadButtonFilename"` attribute back to the CSV download link.

**Review by cstephen — APPROVED** [review]

**Pull request merged**